frontend/src/views/dev/Controls.vue

```diff
@@ -447,6 +447,9 @@ export default {
     profileUri () {
       return this.api.get().profile()._meta.self
     },
+    userUri () {
+      return this.api.get().profile().user()._meta.self
+    },
```

Comment:
Nur zur Info: Mit hal-json-vuex 2.0.0-alpha.6 können wir das dann auch so schreiben:
```suggestion
    profileUri () {
      return this.api.get().$href('profile')
    },
    userUri () {
      return this.api.get().profile().$href('user')
    },
```